packages/tomls/src/token/tokens/susde.toml

```diff
@@ -0,0 +1,70 @@
+[var.susde_vars]
+susdeTokenName = "Ethena sUSDe"
+susdeTokenSymbol = "sUSDE"
```

Comment:
small point, but can we double check and make sure the symbol and token name match what you'd see on other networks?

Reply:
[1f88029](https://github.com/Reya-Labs/reya-deployments/pull/101/commits/1f8802955b18203ba9032499f04afcdf404c4878)